package.json

```diff
@@ -58,10 +67,40 @@
         "./packages/content:build:dev"
       ]
     },
+    "build-and-check:dev": {
+      "dependencies": [
+        "build:dev",
+        "check"
+      ]
+    },
+    "build-and-check:prod": {
+      "dependencies": [
+        "build:prod",
+        "check"
+      ]
+    },
     "check": {
       "dependencies": [
-        "./packages/client:check:types"
+        "./packages/client:check",
+        "check:lint",
+        "check:format"
       ]
+    },
+    "check:lint": {
+      "command": "eslint --color --cache --cache-location .eslintcache \"packages/client/src/**/*.ts\"",
+      "files": [
+        ".eslintrc.json",
+        "packages/client/src/**/*.ts"
+      ],
+      "output": []
```

Comment:
Should `.eslintcache` be included in output?

Reply:
Nope, that would cause it to be deleted every time, making it useless. Unlike with `.tsbuildinfo`, we don't have the problem where deleting an input file leaves the artifact around, because this doesn't remove anything. That's the reason we put `.tsbuildinfo` in the `output` usually.